### DISCUSSION TITLE
Scoop update merge aborts after Powershell process killed by Terminal update by MS Store

### DISCUSSION THREAD

**Discussion by southisup** [ScoopInstaller/Scoop#5123] [Help and Questions] [answered]
I unwisely trusted Microsoft Store to politely queue app updates until affected apps were not running. Instead it updated Windows Terminal immediately, killing its Powershell instance in which `scoop update` was running, resulting in the following error when I try again:
`Updating 'main' bucket...`
`error: Your local changes to the following files would be overwritten by merge:`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/boundary.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/chezmoi.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/circleci-cli.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/cloud-nuke.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/cockroachdb.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/cosign.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/dafny.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/dart.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/docker-compose.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/dolt.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/dvc.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/flatc.json`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`       bucket/geth.json`
`Please commit your changes or stash them before you merge.`
`Aborting`
`Scoop was updated successfully!`

On Windows 10

Further `scoop update` attempts succeed for `extras` bucket, but fail again as above for `main` bucket. Past tickets for similar issues all seem to be closed with varying solutions. How can I repair my install of scoop, and is someone able to lodge an appropriate bug report? I don't understand the problem well, and don't have any further information beyond the above, sorry.

**Comment by rashil2000** [comment, marked as answer]
Run this:
```
scoop bucket rm main
scoop bucket add main
```

**Reply by southisup** [reply]
Thanks! I wasn't confident to try such a thing, in case it broke existing apps.
